src/hooks/useFarming.ts

```diff
@@ -215,6 +214,6 @@ export function useFarming({ lpContract, stakingContract }: UseFarming = {}): Fa
   return {
     farmApr,
     lpApr,
-    totalApr
+    totalApr: new BigNumber(farmApr).plus(lpApr).toString()
```

Comment:
this has to recompute on every re-render of any component that uses this hook

Reply:
also in this is more of a consistency thing:
we use `bn` imported from `utils/math`

Reply:
I think if you just switch to `bn` for consistency and set the totalApr back to state that will be fine

Reply:
Yea or you could use a useMemo as well to make sure that it did not rerender unless the aprs changed.

Reply:
We all talked about it and decided to go with a useMemo on the totalApr